envoy/upstream/upstream.h

```diff
@@ -766,6 +766,9 @@ class PrioritySet {
   GAUGE(upstream_cx_tx_bytes_buffered, Accumulate)                                                 \
   GAUGE(upstream_rq_active, Accumulate)                                                            \
   GAUGE(upstream_rq_pending_active, Accumulate)                                                    \
+  GAUGE(upstream_rq_retry_active, Accumulate)                                                      \
+  GAUGE(upstream_rq_retry_backoff_exponential_active, Accumulate)                                  \
+  GAUGE(upstream_rq_retry_backoff_ratelimited_active, Accumulate)                                  \
```

Comment:
High-level comment:
Gauges have non-negligible overhead, and I wonder whether these (or all of these) need to be added as gauges or just tracked as internal state variables.
Can you provide some clarification what's the added value of these gauges?